lib/puppet/pops/lookup/hiera_config.rb

```diff
@@ -106,7 +106,18 @@ def self.config_exist?(config_root)
   def self.symkeys_to_string(struct)
     case(struct)
     when Hash
-      Hash[struct.map { |k,v| [k.to_s, symkeys_to_string(v)] }]
+      map = struct.map { |k,v|
+        case k.class.to_s
+        when "Symbol"
+          key = k.to_s
+        else
+          key = k
+        end
+        value = symkeys_to_string(v)
+        [key, value]
+      }
+
+      Hash[map]
```

Comment:
Ping @thallgren, not sure where exctly the `symkeys_to_string` is used - by its location it suggests that it is for hiera's configuration - but it seems to affect data. For the hiera.yaml config I wonder if it should keep `Integer` keys (the changed version would do that).